examples/example01.toit

```diff
@@ -1,13 +1,14 @@
+// Copyright (c) 2021 Ekorau LLC
```

Comment:
We generally like to make examples BSD0, as they are often copied, and we don't want users to deal with copyright statements then. But that's up to you.